docs/reference/objects/product/extra.md

```diff
@@ -77,6 +87,8 @@ number
 {: .label .fs-1 }
 
 The initial stock for the extra, if the extra has unlimited inventory this will return `nil`.
+If this extra is accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return the initial stock for the extra on the given date.
+If this extra is accessed independently or through a product, then it will return the initial stock for the default extra, managed in **Experience > Extras**.
```

Comment:
Should this now be updated based on this thread? https://easolhq.slack.com/archives/C03AJ5F6L1L/p1689348944793599